src/components-examples/cdk/stepper/cdk-linear-stepper-with-form/cdk-linear-stepper-with-form-example.html

```diff
@@ -0,0 +1,17 @@
+<example-custom-linear-stepper [linear]="isLinear">
+  <cdk-step [stepControl]="firstFormGroup">
+    <label>Step 1 input</label>
```

Comment:
Is there something going on behind the scenes to associate this label to the form control for accessibility purposes?

Reply:
Nope, nothing automatic. @nielsr98 the `<label>` should have a `for` attribute pointing to the `<input>` element (which will then need an ID), or you can alternative wrap the `<input>` inside the `<label>`

Reply:
Updated to use a `for` attribute.